Document_Device.py

```diff
@@ -1,134 +1,184 @@
 #$language = "python"
 #$interface = "1.0"
 
-# Script to commit the configuration of an IOS device, and log the output
-# of some common commands for later reference
+# Document_Device.py
+# 
+# Description:
+#   Sends a series of Cisco Show commands one by one as listed in the
+#   COMMANDS array.  The results of each command are captured into a
+#   variable, and then written to an individual log file (one log file
+#   for each command).
+# 
+#   Filename format is:
+#   ~/$savepath/<Host Name>-<Command Name>-<Date Format>.txt
 
 import os
 import subprocess
 import datetime
 import sys
 
-SCRIPT_TAB = crt.GetScriptTab()
-
-TAB_NAME = SCRIPT_TAB.Caption
+# Adjust these to your environment
+savepath = 'Configs/'
+mydatestr = '%Y-%m-%d-%H-%M-%S'
 
 COMMANDS = [
-	"show ip interface brief",
-	"show ipv6 interface brief",
-	"show cdp neighbors",
+	"show access-list",
+	"show call active voice brief",
+	"show cann history voice brief",
```

Comment:
Can you propose changes to someone elses pull requests through git without making a new request?

Noticed I've typo'd call here